app/javascript/flavours/glitch/features/local_settings/page/index.js

```diff
@@ -76,6 +76,14 @@ export default class LocalSettingsPage extends React.PureComponent {
           >
             <FormattedMessage id='settings.side_arm' defaultMessage='Secondary toot button:' />
           </LocalSettingsPageItem>
+          <LocalSettingsPageItem
+            settings={settings}
+            item={['side_arm_auto']}
+            id='mastodon-settings--side_arm_auto'
+            onChange={onChange}
```

Comment:
It would be cool if this item were only enabled when the secondary toot button is set to anything else than `None`.
Unfortunately, that requires more logic than what's defined for other items. I can have a look later if you want.